### ISSUE TITLE
Error when I try to install

### ISSUE THREAD

**Issue by Dude4Linux** [JedMeister/jessie-lxc-dev#1]
@JedMeister, When I tried to install your version of lxc-dev on a freshly installed LXC appliance, I get the following error.  Looks like the package still includes part of lxc.

```
# dpkg -i lxc-dev-tkl_1.0.6-6+deb8u1+turnkey1_amd64.deb 
Selecting previously unselected package lxc-dev-tkl.
(Reading database ... 28894 files and directories currently installed.)
Preparing to unpack lxc-dev-tkl_1.0.6-6+deb8u1+turnkey1_amd64.deb ...
Unpacking lxc-dev-tkl (1:1.0.6-6+deb8u1+turnkey1) ...
dpkg: error processing archive lxc-dev-tkl_1.0.6-6+deb8u1+turnkey1_amd64.deb (--install):
 trying to overwrite '/etc/init.d/lxc', which is also in package lxc 1:1.0.6-6+deb8u1
dpkg-deb: error: subprocess paste was killed by signal (Broken pipe)
Processing triggers for systemd (215-17+deb8u2) ...
Errors were encountered while processing:
 lxc-dev-tkl_1.0.6-6+deb8u1+turnkey1_amd64.deb
root@lxc ~# dpkg -l lxc
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version      Architecture Description
+++-==============-============-============-=================================
ii  lxc            1:1.0.6-6+de amd64        Linux Containers userspace tools
```

**Comment by JedMeister** [comment]
Ok looks like I must have broken something when I tweaked the source package so it would only build the `lxc-dev`(`-tkl`) package. I must admit that I only did basic testing on that one.

Also it looks like it depends on a turnkey version of lxc too (which is no good as there isn't a turnkey lxc package...)

I'll get back to this by the end of the week hopefull. It should be a fairl;y easy fix and I will test it better this time.

In the meantime, the original one I produced (and included) should work. Try this one: https://github.com/JedMeister/jessie-lxc-dev/blob/97457612684ef4fc1717936b600b260cbcf96fbe/lxc-dev-tkl_1.0.6-6+deb8u1+turnkey1_amd64.deb?raw=true
